src/data/finset/pointwise.lean

```diff
@@ -134,7 +134,7 @@ coe_injective $ by rw [coe_preimage, set.inv_preimage, coe_inv]
 
 end has_involutive_inv
 
-/-! ### Finset addition/multiplication -/
+/-! ### Scalar subtraction of finsets -/
```

Comment:
Huh? This section begins with `has_mul`...

Reply:
Whoops, I meant to change the next one.